app/jobs/dttp/sync_users_job.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Dttp
+  class SyncUsersJob < ApplicationJob
+    queue_as :dttp
+
+    def perform(path = nil)
```

Comment:
Can we split this is up a bit to remove the duplication on 14?

maybe added some private methods like:

```
def user_list
  ...
end

def next_page_url
  ...
end

def enqueue_next_page
   ...
end
```

Reply:
done